tests/tests/operation_tests.cpp

```diff
@@ -2336,6 +2336,347 @@ BOOST_AUTO_TEST_CASE( vesting_balance_withdraw_test )
+
+      ACTORS((paul)(michael)(rachel)(alice));
+
+      // create assets
+      const auto& bitusd = create_bitasset("USDBIT", paul_id);
+      const auto& core   = asset_id_type()(db);
+      asset_id_type bitusd_id = bitusd.id;
+      asset_id_type core_id = core.id;
+
+      // fund accounts
+      transfer(committee_account, michael_id, asset( 100000000 ) );
+      transfer(committee_account, paul_id, asset(10000000));
+      transfer(committee_account, alice_id, asset(10000000));
+
+      // allow global settle in bitusd
+      asset_update_operation op;
+      op.issuer = bitusd.issuer;
+      op.asset_to_update = bitusd.id;
+      op.new_options.issuer_permissions = global_settle;
+      op.new_options.flags = bitusd.options.flags;
+      op.new_options.core_exchange_rate = price({asset(1,asset_id_type(1)),asset(1)});
+      trx.operations.push_back(op);
+      sign(trx, paul_private_key);
+      PUSH_TX(db, trx);
+      generate_block();
+      trx.clear();
+
+      // add a feed to asset
+      update_feed_producers( bitusd, {paul.id} );
+      price_feed current_feed;
+      current_feed.maintenance_collateral_ratio = 1750;
+      current_feed.maximum_short_squeeze_ratio = 1100;
+      current_feed.settlement_price = bitusd.amount( 100 ) / core.amount(5);
+      publish_feed( bitusd, paul, current_feed );
+
+      BOOST_CHECK_EQUAL(get_balance(paul_id(db), core), 10000000);
+
+      // paul gets some bitusd
+      auto call_paul = *borrow( paul_id(db), bitusd_id(db).amount(1000), core_id(db).amount(100));
+      call_order_id_type call_paul_id = call_paul.id;
+      BOOST_REQUIRE_EQUAL( get_balance( paul_id(db), bitusd_id(db) ), 1000 );
+
+      // and transfer some to rachel
+      transfer(paul_id, rachel_id, asset(200, bitusd_id));
+
+      BOOST_CHECK_EQUAL(get_balance(rachel_id(db), core_id(db)), 0);
+      BOOST_CHECK_EQUAL(get_balance(rachel_id(db), bitusd_id(db)), 200);
+      BOOST_CHECK_EQUAL(get_balance(michael_id(db), bitusd_id(db)), 0);
+      BOOST_CHECK_EQUAL(get_balance(michael_id(db), core_id(db)), 100000000);
+
+      // michael selling core
+      auto call_michael = *borrow(michael_id(db), bitusd_id(db).amount(6), core_id(db).amount(8));
+      call_order_id_type call_michael_id = call_michael.id;
+
+      // add global settle
+      force_global_settle(bitusd_id(db), bitusd_id(db).amount(10) / core_id(db).amount(1));
+      generate_block();
+
+      BOOST_CHECK_EQUAL(get_balance(rachel_id(db), core_id(db)), 0);
+      BOOST_CHECK_EQUAL(get_balance(rachel_id(db), bitusd_id(db)), 200);
+      BOOST_CHECK_EQUAL(get_balance(michael_id(db), bitusd_id(db)), 6);
+      BOOST_CHECK_EQUAL(get_balance(michael_id(db), core_id(db)), 100000000);
+      BOOST_CHECK_EQUAL(get_balance(paul_id(db), core_id(db)), 9999900);
+      BOOST_CHECK_EQUAL(get_balance(paul_id(db), bitusd_id(db)), 800);
+
+      // all call orders are gone after global settle
+      BOOST_CHECK( db.find_object(call_paul_id) == nullptr);
```

Comment:
`db.find_object(call_paul_id) == nullptr` can be replaced with `!db.find_object(call_paul_id)`. Not a big deal though.